src/main/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandController.java

```diff
@@ -65,4 +70,76 @@ public RichMessage onReceiveSlashCommandKeeperAdd(@RequestParam("token") String
+
+        logger.debug("Received slash command KeeperDismiss: from user: [{}] command: [{}] token: [{}]",
+                fromUser, text, token);
+
+        if (!token.equals(slackToken)) {
+            logger.warn("Received invalid slack token: [{}] in command KeeperDismiss for user: [{}]", token, fromUser);
+            return new RichMessage("Sorry! You're not lucky enough to use our slack command.");
+        }
+
+        logger.debug("Started create slackParsedCommand and create keeper request");
+        SlackParsedCommand slackParsedCommand = slackNameHandlerService.createSlackParsedCommand(fromUser, text);
+        KeeperRequest keeperRequest = new KeeperRequest(slackParsedCommand.getFromUser().getUuid(),
+                                                        receiveToUser(slackParsedCommand).getUuid(),
+                                                        receiveToDirections(slackParsedCommand));
+
+        logger.debug("Finished create slackParsedCommand and create keeper request");
+
+        String[] result = keeperService.sendKeeperDismissRequest(keeperRequest);
+        logger.debug("Received response from Keeper service: [{}]", Arrays.toString(result));
+
+        String response = "ERROR. Something went wrong. Keeper was not dismissed :(";
+
+        if (result.length > 0) {
+            response = String.format("Keeper: %s in direction: %s dismissed" ,
+                    slackParsedCommand.getFirstUser().getSlack(), keeperRequest.getDirection());
+        }
+
+        logger.info("Keeper command processed : user: [{}] text: [{}] and sent response into slack: [{}]",
+                fromUser, text, response);
+
+        return new RichMessage(response);
+    }
+
+    private UserDTO receiveToUser(SlackParsedCommand slackParsedCommand) {
+
+        int userCount = slackParsedCommand.getUserCount();
+
+        if (userCount > 1) {
+            throw new WrongCommandFormatException(String.format("We found %d slack names in your command: '%s' " +
+                            "You can not perform actions with several slack names.",
+                    slackParsedCommand.getUserCount(), slackParsedCommand.getText()));
+        }
+
+        if (userCount == 0) {
+            throw new WrongCommandFormatException(String.format("We didn't find slack name in your command. '%s' " +
+                    "You must write user's slack name to perform action with keepers.", slackParsedCommand.getText()));
+        }
+
+        return slackParsedCommand.getFirstUser();
+    }
+
+    private String receiveToDirections(SlackParsedCommand parsedCommand){
+
+        String textWithoutSlackNames = parsedCommand.getTextWithoutSlackNames();
+
+        if (textWithoutSlackNames.length() == 0){
+            throw new WrongCommandFormatException(String.format("We didn't find direction in your command: '%s' " +
+                    "You must write direction to perform action with keepers.", parsedCommand.getText()));
```

Comment:
We didn't find any direction in your command
You must write the direction to perform the action with keepers